en-US/Characters_Codex/Credits/Artists.md

```diff
@@ -1,6 +1,10 @@
 ## Artists
 
+<!-- spell-checker:disable -->
```

Comment:
This is done preemptively, since I except the spell checker will not know the names of at least some of the artists. :wink: